Chocolatey-AU.build.ps1

```diff
@@ -0,0 +1,375 @@
+param(
+    [string]
+    $OutputDirectory = "$PSScriptRoot/Output",
```

Comment:
🤔 Not sure why I didn't pick `code_drop`. Perhaps because this was a PowerShell module. But yeah, should be consistent about where we output things. I'll get that updated.